### PR TITLE
Bump gl-native 10.9.0, common 23.1.1.

### PR DESCRIPTION
**Pull request by yunikkk**
## Features ✨ and improvements 🏁
* Updated mapbox-common to v23.1.1.
* Introduced `location-indicator-opacity` to location indicator layer that allows to control layer opacity.
* Rendering performance improvements for fast paced camera changes

## Bug fixes 🐞
* Fixed flickering of vertically shifted line labels on style change, caused by the wrong initial placement.
* Fixed location indicator rendering as a rectangle on low zoom levels.
* Fixed crash caused by using of invalid paint property binders.
* Fixed "phantom tiles" artefacts after GeoJSON source update when asynchronous tile loading is enabled.
* Mitigated symbol flickering on zooming out globe map.
* Fixed random crash on tiles update when asynchronous uploading is enabled.
* Fixed pixel flickering between tiles with darker styles in globe view.
* Location indicator layer now occluded by terrain to align rendering behavior with other location indicator rendering types.
* Fixed symbol flickering after runtime styling caused by using outdated symbol placement on updated symbol buckets.
* Allow Light intensity and Light shadow intensity to accept expressions when using setStyleLight API call.
* Fixed a bug in cameraForGeometry when called on a map with padding set.

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [x] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,29 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Fix "phantom tiles" artefacts after GeoJSON source update when asynchronous tile loading is enabled. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Mitigate symbol flickering on zooming out globe map. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix random crash on tiles update when asynchronous uploading is enabled. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix pixel flickering between tiles with darker styles in globe view. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Textures of raster tiles and location indicator are now recovered after Metal command buffer errors. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Location indicator layer now occluded by terrain to align rendering behavior with other location indicator rendering types. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fixe symbol flickering after runtime styling caused by using outdated symbol placement on updated symbol buckets. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
```
[the remark this answers is not in the document]
Comment (yunikkk) [reply CHANGELOG.md]: Nice! Thanks

**Review by ank27 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,29 @@ Mapbox welcomes participation and contributions from everyone.
 
 # 10.9.0
 
+## Features ✨ and improvements 🏁
+* Introduce `location-indicator-opacity` to location indicator layer that allows to control layer opacity. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Introduce maps Version API. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
```
Comment (yunikkk) [inline CHANGELOG.md:11]: Seems this one is not relevant for customers

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,9 +6,29 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Rendering performance improvements for fast paced camera changes ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Add `line-border-width` and `line-border-color` experimental line layer properties. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
```
Comment (yunikkk) [inline CHANGELOG.md:13]: Should we leave this one as well?

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,9 +6,29 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Fix flickering of vertically shifted line labels on style change, caused by the wrong initial placement. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix location indicator rendering as a rectangle on low zoom levels. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix crash caused by using of invalid paint property binders. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix "phantom tiles" artefacts after GeoJSON source update when asynchronous tile loading is enabled. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Mitigate symbol flickering on zooming out globe map. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix random crash on tiles update when asynchronous uploading is enabled. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Fix pixel flickering between tiles with darker styles in globe view. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
+* Textures of raster tiles and location indicator are now recovered after Metal command buffer errors. ([1760](https://github.com/mapbox/mapbox-maps-android/pull/1760))
```
[the remark this answers is not in the document]
Comment (yunikkk) [reply CHANGELOG.md:24]: Right, removing

**Review by pengdev — APPROVED** [review]

**Review by kiryldz — CHANGES_REQUESTED** [review]
Now we have our Android SDK delta between rc and stable in the changelog but we have full gl-native changelog (including changes for beta, rc and stable). We need to align it - either add all our SDK changes here as well or understand what's the gl-native delta between rc and stable.

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,8 +6,52 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Fix scale bar text overlapping. ([1728](https://github.com/mapbox/mapbox-maps-android/pull/1728))
+* Fix issue where `Widget` is not immediately rendered when added to the `MapView`. ([1708](https://github.com/mapbox/mapbox-maps-android/pull/1708))
+* Mitigate symbol flickering on zooming out globe map. ([1729](https://github.com/mapbox/mapbox-maps-android/pull/1729))
+* Fix random crash on tiles update when asynchronous uploading is enabled. ([1729](https://github.com/mapbox/mapbox-maps-android/pull/1729))
+* Fix a crash when Google location service is started in background but stopped in foreground. ([1729](https://github.com/mapbox/mapbox-maps-android/pull/1729))
+* Fix an issue where mock field is missing from the LocationEngine updates. ([1729](https://github.com/mapbox/mapbox-maps-android/pull/1729))
+* Make telemetry a single instance. Avoids module recreation when coming from background. ([#1695](https://github.com/mapbox/mapbox-maps-android/pull/1695))
+* Fix an issue where location engine doesn't produce bearing/accuracy/speed information with the location updates. ([1696](https://github.com/mapbox/mapbox-maps-android/pull/1696))
```
Comment (kiryldz) [inline CHANGELOG.md:35]: not needed as those fixes were relevant between beta and rc and not between stable versions

**Review by kiryldz — APPROVED** [review]
LGTM with the comment

File: CHANGELOG.md (hunk 5 above)
Comment (yunikkk) [reply CHANGELOG.md:35]: All righty

**Review by kiryldz — APPROVED** [review]

**Review by pengdev — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,10 +4,51 @@ Mapbox welcomes participation and contributions from everyone.
 
 # main
 
-# 10.9.0
+# 10.9.0 October 21, 2022
```
Comment (pengdev) [inline CHANGELOG.md:7]: nit: release date will be automatically added by release train app, and the PR will be automatically updated to both main and release branch.

Comment (yunikkk) [reply CHANGELOG.md]: y

**Review by ank27 — APPROVED** [review]

**Pull request merged**